### PR TITLE
Post 1.1.0 version bumps

### PR DESCRIPTION
**Pull request by msimberg**
Updates versions and files. Makes some changes to the release procedures again.

### CODE REVIEW

File: hpx/config/version.hpp [hunk 1]
```diff
@@ -26,19 +26,19 @@
...
 //
-#define HPX_VERSION_FULL         0x010100
+#define HPX_VERSION_FULL         0x010200
 
 #define HPX_VERSION_MAJOR        1
 #define HPX_VERSION_MINOR        1
 #define HPX_VERSION_SUBMINOR     0
 
-#define HPX_VERSION_DATE         20180324
+#define HPX_VERSION_DATE         20180919
```
Comment (hkaiser) [inline hpx/config/version.hpp]: Nice try!

Comment (msimberg) [reply hpx/config/version.hpp]: We'll see ;)

File: hpx/config/version.hpp [hunk 2]
```diff
@@ -26,19 +26,19 @@
 //  HPX_VERSION_DATE   YYYYMMDD is the date of the release
 //                               (estimated release date for master branch)
 //
-#define HPX_VERSION_FULL         0x010100
+#define HPX_VERSION_FULL         0x010200
 
 #define HPX_VERSION_MAJOR        1
 #define HPX_VERSION_MINOR        1
```
Comment (hkaiser) [inline hpx/config/version.hpp]: I think you missed this one...

Comment (msimberg) [reply hpx/config/version.hpp]: Yep, thanks!

File: CMakeLists.txt [hunk 3]
```diff
@@ -42,7 +42,7 @@ endif()
 project(HPX CXX C)
 
 set(HPX_MAJOR_VERSION 1)
-set(HPX_MINOR_VERSION 1)
+set(HPX_MINOR_VERSION 2)
 set(HPX_PATCH_LEVEL   0)
 set(HPX_VERSION "${HPX_MAJOR_VERSION}.${HPX_MINOR_VERSION}.${HPX_PATCH_LEVEL}")
 set(HPX_LIBRARY_VERSION "${HPX_VERSION}")
```
Comment (hkaiser) [inline CMakeLists.txt]: In the future we might want to generate our version.hpp content from these definitions, that should simplify things a bit...

**Pull request merged**